MainExp-analyses-Native.Rmd

```diff
@@ -433,23 +639,42 @@ near = glmer(nativeAccuracy ~ group +
 
 summary(near)
 
+Anova(near)
+
+near_EB <- subset(allData, meaning == "near" & group == 'EB')
+binom.test(sum(near_EB$grade), nrow(near_EB), p = .5)
+
+near_SC <- subset(allData, meaning == "near" & group == 'SC')
+binom.test(sum(near_SC$grade), nrow(near_SC), p = .5)
+
 ```
 
+
```

Comment:
As well here, I would move the plot into another script in which we also run summary general statistics (namely group accuracy percentage etc)

Reply:
The summary + plot would be a comprehensive one for native / raw / experiment tool etc.